### PR TITLE
Update splunk forwarder role so it can be installed on Amazon Linux OS

### PR DESCRIPTION
**Pull request by schenedx**
Configuration Pull Request
---

Make sure that the following steps are done before merging:

  - [ ] A DevOps team member has approved the PR.
  - [x] Are you adding any new default values that need to be overridden when this change goes live? If so:
    - [x] Add an entry to the CHANGELOG.
  - [ ] If you are making a complicated change, have you performed the proper testing specified on the [Ops Ansible Testing Checklist](https://openedx.atlassian.net/wiki/display/EdxOps/Ops+Ansible+Testing+Checklist)?  Adding a new variable does not require the full list (although testing on a sandbox is a great idea to ensure it links with your downstream code changes).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -81,6 +81,7 @@
 
 - Role: server_utils
```
Comment (feanil) [inline CHANGELOG.md]: Changelog entries should go at the top of the file.

**Review by feanil — CHANGES_REQUESTED** [review]
@schenedx one small change needed otherwise looks good.

File: CHANGELOG.md (hunk 1 above)
Comment (schenedx) [reply CHANGELOG.md]: @feanil This is one address.

**Comment by schenedx** [comment]
@feanil I addressed your feedback. Please approve and merge.

File: CHANGELOG.md (hunk 1 above)
Comment (feanil) [reply CHANGELOG.md]: @schenedx generally new entries in a changelog should go at the top of the changelog.  This makes it easier to see what's changed recently.  The line below and the two lines about the splunkforwarder should all go to the top of this file.

Comment (schenedx) [reply CHANGELOG.md]: @feanil Gotcha. Made the updates. Please review again

**Review by feanil — APPROVED** [review]

File: playbooks/roles/splunkforwarder/tasks/main.yml [hunk 2]
```diff
@@ -51,7 +67,7 @@
     state: present
     append: yes
     groups: syslog,adm
-  when: download_deb.changed
+  when: download_package.changed
```
Comment (jibsheet) [inline playbooks/roles/splunkforwarder/tasks/main.yml:70]: On an ubuntu box, the second declaration of download_deb for rpm paves over the one set for the deb download. (even though you have when,  ansible still paves over the register for you)

This empty download_deb means that splunk's user is never created, app is never started, etc.

This is breaking build from scratches for me https://gocd.tools.edx.org/go/tab/build/detail/stage-xqueue/16/build_ami/1/stage_edx

Comment (schenedx) [reply playbooks/roles/splunkforwarder/tasks/main.yml:70]: This is now fixed forward with https://github.com/edx/configuration/pull/4427

**Pull request merged**